website/server/models/task.js

```diff
@@ -169,9 +169,14 @@ TaskSchema.statics.findByIdOrAlias = async function findByIdOrAlias (identifier,
 // Sanitize user tasks linked to a challenge
 // See http://habitica.wikia.com/wiki/Challenges#Challenge_Participant.27s_Permissions for more info
 TaskSchema.statics.sanitizeUserChallengeTask = function sanitizeUserChallengeTask (taskObj) {
-  let initialSanitization = this.sanitize(taskObj);
-
-  return _.pick(initialSanitization, ['streak', 'checklist', 'attribute', 'reminders', 'tags', 'notes', 'collapseChecklist', 'alias', 'yesterDaily', 'counterDown', 'counterUp']);
+  const initialSanitization = this.sanitize(taskObj);
+
+  return _.pick(initialSanitization, [
+    'streak', 'checklist', 'attribute', 'reminders',
+    'tags', 'notes', 'text', 'collapseChecklist',
+    'alias', 'yesterDaily', 'counterDown', 'counterUp',
+    'up', 'down',
```

Comment:
is it necessary to add the up and down to this whitelist?

Reply:
at first I thought this would fix all the issues, but it is by-design to not allow changes to `text` / `up`/`down` and I assumed it was allowed, so when I wrote the tests (these sanitization weren't available back then) it worked right away.

Now that I revert this change, removed all others changes of text,up,down and reverted the sanitization conditions, all tests works fine again.